profiles/views.py

```diff
@@ -92,13 +92,14 @@ def submit_profiles(request):
                                 profile=stored_profile,
                                 identifier=payload.get('PayloadIdentifier', ''),
                                 uuid=payload.get('PayloadUUID', ''),
-                                payload_type=payload.get('PayloadType', [])
+                                payload_type=payload.get('PayloadType', '')
                             )
 
                             if utils.is_postgres():
                                 payloads_to_save.append(payload_item)
                             else:
                                 payload_item.save()
+                            break
```

Comment:
This needs to be outdented one block level I think.

Reply:
◀️

Reply:
Wrong, two levels. It's to break out of the `for profile in profiles_list:` loop that is looking for the Profile item  that matches the DB Profile item.

Reply:
As it stands, this would break out of the for loop creating the payload contents, creating only one before moving onto the next profile.